src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const sum = 40 * days;
+
+  if (sum < 120) {
+    return sum;
```

Comment:
Avoid using `else if` after an `if` statement that contains a `return`. The `else` is not necessary because if the `if` condition is true, the function will return and the subsequent code will not execute.